test/glimesh_web/controllers/user_registration_controller_test.exs

```diff
@@ -40,7 +40,7 @@ defmodule GlimeshWeb.UserRegistrationControllerTest do
       conn = get(conn, "/")
       response = html_response(conn, 200)
       assert response =~ username
-      assert response =~ "\nSettings</a>"
+      assert response =~ "\nYour Settings</a>"
```

Comment:
Trying to get away from "Your x", can we change this back?

Reply:
I just changed the test since someone else change it backed to Your Settings in the navbar.

Reply:
It's most likely due to me importing the English translation file into the project. I'll take a look though.